test/fixtures/wallet-web/docker-compose-web.yml

```diff
@@ -31,9 +31,7 @@ services:
       - EDV_CLEAR_CACHE=10m
       - USE_EDV_BATCH=true
       - STATIC_ASSETS=/etc/static
-      # To add static images, set STATIC_ASSETS_URL=https://wallet.trustbloc.local:8091,
-      # then add images under /test/fixtures/wallet-web/static/images
-      - STATIC_ASSETS_URL=
+      - STATIC_ASSETS_URL=https://wallet.trustbloc.local:8091
```

Comment:
name is bit ambiguous

Reply:
This name was existing from before, it refers to mounted static folder which has config/images etc inside, I think this is apt name for it, what else would you suggest